source/extensions/tracers/xray/tracer.cc

```diff
@@ -179,20 +191,31 @@ void Span::setTag(absl::string_view name, absl::string_view value) {
   }
 
   if (name == HttpUrl) {
-    http_request_annotations_.emplace(SpanUrl, value);
+    http_request_annotations_.emplace(SpanUrl, ValueUtil::stringValue(std::string(value)));
```

Comment:
`value` is a string already.
so this should be `ValueUtil::stringValue(value);` instead?

Reply:
So value is a string_view, and based off my read of the [abseil docs](https://abseil.io/docs/cpp/guides/strings) and the header there is no implicit conversion from string_view to string.  As rusty as I am at c++, I'm assuming the reason the "implicit conversion" works for flat_hash_map::emplace is because its constructing from arguments internally

Reply:
if value is a string_view, then this is fine.